File: JS/URI/1080.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');
let maior = 0;
let posicao = 0;
for (var i = 1; i < 101; i++) {

    

    let number = parseInt(lines.shift());

    if(number > maior)
    {
        maior = number;
        posicao = i;
    }
    

}

console.log(maior+"");
console.log(posicao+"");

File: C/URI/1080.c
#include <stdio.h>
 
int main() {
 
  int numero, n, maior, posicao;
        n = 0;
        maior = 0;
        while(n < 100)
        {
        scanf("%d", &numero);
                if(numero > maior)
                {
                        maior = numero;
                        posicao = n+1;
                }
        n = n + 1;
        }
        printf("%d\n%d\n", maior, posicao);

    return 0;
}